pkg/nsx/services/nsxserviceaccount/store.go

```diff
@@ -40,17 +41,17 @@ type ClusterControlPlaneStore struct {
 }
 
 func (s *ClusterControlPlaneStore) Apply(i interface{}) error {
-	pis := i.(*[]model.ClusterControlPlane)
-	for _, pi := range *pis {
-		if pi.MarkedForDelete != nil && *pi.MarkedForDelete {
-			err := s.Delete(pi)
-			log.V(1).Info("delete PI from store", "pi", pi)
+	ccps := i.(*[]model.ClusterControlPlane)
+	for _, ccp := range *ccps {
+		if ccp.MarkedForDelete != nil && *ccp.MarkedForDelete {
+			err := s.Delete(ccp)
+			log.V(1).Info("delete ClusterCP from store", "ClusterCP", ccp)
 			if err != nil {
 				return err
 			}
 		} else {
-			err := s.Add(pi)
-			log.V(1).Info("add PI to store", "pi", pi)
+			err := s.Add(ccp)
+			log.V(1).Info("add ClusterCP to store", "ClusterCP", ccp)
```

Comment:
&ccp ?

Reply:
Changed these 2 functions to VPC format.